apps/eo_web/src/api/useApi.ts

```diff
@@ -109,31 +109,41 @@ export const useApi = () => {
   };
 
   const eligibleEmail = async (email: string) => {
-    return await api.get<{ success: boolean; message: string }>(
-      `${API_ELIXIR}/v2/profiles/eligible?email=${email}`,
-      authHeader,
-    );
+    return await api.get<{
+      success: boolean;
+      message: string;
+    }>(`${API_ELIXIR}/v2/profiles/eligible?email=${email}`, authHeader);
   };
```

Comment:
Email should be sent with `params` option. Response object could be outsourced and reused. Use arrow function advantage.

In another file:
```
export type ApiResponse = {
   success: boolean;
   message: string;
}
```

Then:
```suggestion
  const eligibleEmail = async (email: string) =>  await api.get<{
      success: boolean;
      message: string;
    }>(`${API_ELIXIR}/v2/profiles/eligible`. {
      ...authHeader,
      params: { email }
    });
```

Reply:
I would personally go one step further and attach the authHeaders and baseUrl to a single axios instance to avoid repeating yourself in every function, something like this:
```
const api = axios.create({
  baseURL: API_ELIXIR,
  headers: authHeader
});
```

Then this function would just be:
````
const eligibleEmail = async (email: string) =>  await api.get<ApiResponse>(`/v2/profiles/eligible`. {  params: { email } });
````

Reply:
Good suggestion

Reply:
@hyanez-lightit  thank you for your review, these changes were made in another PR that I was working on. that was. good review.